### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ReikutsuNeko**
Tested the following app using WSA Version 2204.40000.20.0 (updated through windows update):
- Arknights (EN)
- Azure Lane (EN)
- ウマ娘 プリティーダービー (Uma Musume PrettyDerby)
- Fire Emblem Heroes
- Girls Frontline (EN)
- Fate/Grand Order (EN)

Graphics using NVIDIA GeForce GTX 1050 Ti Mobile on Windows 11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,20 +94,20 @@ wsa://com.android.settings
...
 | AquaMail (Pro) | 1.34.0-118 | 11 | ✅
 | Arcaea | 3.8.8 | 11 | ⚠️ | Keyboard doesn't work on login/register form
-| Arknights | 5.0.01 | 11 | 🆖 | Can't login using Google account. Unstable FPS throught the game, especially low FPS in combat for AMD system PC
+| Arknights | 10.0.01 | 11, 12 | 🆖 | Can't login using Google account. Unstable FPS throught the game, especially low FPS in combat for AMD system PC. Stable FPS throughout the game using Nvidia
 | Asphalt 9 || 11 | ⚠️ | Keyboard unsupported
 | Audible | 3.15.0 | 11 | ✅
 | Aurora Store | 4.1.1 | 12, 11 | ✅
 | Authy | 24.8.5 (139) | 11 | ✅ || Produces warnings about GMS which are safe to ignore.
-| Azur Lane | 6.0.820 | 11 | ⚠️ | Some characters may appear missing and the game can get stuck while in combat, this can be fixed by restarting the app.
+| Azur Lane | 6.1.2 | 11, 12 | ⚠️ | Sometimes stuck on downloading resources, can be fixed by restarting the app. Overall gameplay, got stable FPS using Nvidia
```
Comment (pratyakshm) [inline README.md:102]:
```suggestion
| Azur Lane | 6.1.2 | 12, 11 | ⚠️ | Sometimes stuck on downloading resources, can be fixed by restarting the app. Overall gameplay, got stable FPS using NVIDIA GeForce GTX 1050 Ti Mobile
```

File: README.md [hunk 2]
```diff
@@ -94,20 +94,20 @@ wsa://com.android.settings
 | Aptoide App Store | 9.20.2.1 | 11 | ✅ | Sometimes, downloads might get stuck
 | AquaMail (Pro) | 1.34.0-118 | 11 | ✅
 | Arcaea | 3.8.8 | 11 | ⚠️ | Keyboard doesn't work on login/register form
-| Arknights | 5.0.01 | 11 | 🆖 | Can't login using Google account. Unstable FPS throught the game, especially low FPS in combat for AMD system PC
+| Arknights | 10.0.01 | 11, 12 | 🆖 | Can't login using Google account. Unstable FPS throught the game, especially low FPS in combat for AMD system PC. Stable FPS throughout the game using Nvidia
```
Comment (pratyakshm) [inline README.md:97]:
```suggestion
| Arknights | 10.0.01 | 12, 11 | 🆖 | Can't login using Google account. Unstable FPS throught the game, especially low FPS in combat for AMD system PC. Stable FPS throughout the game using NVIDIA GeForce GTX 1050 Ti Mobile
```

File: README.md [hunk 3]
```diff
@@ -151,9 +151,9 @@ wsa://com.android.settings
 | Facebook Messenger | 330.0.0.12.116 (x86_64) | 11 | ⚠️ | Chat Heads don't work
 | Fancade | 1.7.6 | 11 | ❌ | App crashes
 | FAST Speed Test | 1.0.8 (88) | 11 | ✅
-| Fate/Grand Order (US) FGO | 2.22.1 (135) | 11 | ✅ || A little unstable, but playable
+| Fate/Grand Order (US) FGO | 2.34.0 | 11, 12 | 🆖 || Require Google Play Services, can be skippable if you have Google Play Service (APK) installed
```
Comment (pratyakshm) [inline README.md:154]:
```suggestion
| Fate/Grand Order (US) FGO | 2.34.0 | 12, 11 | 🆖 || Require Google Play Services, skippable if you have Google Play Service (APK) installed
```

File: README.md [hunk 4]
```diff
@@ -94,20 +94,20 @@ wsa://com.android.settings
...
+| Azur Lane | 6.1.2 | 11, 12 | ⚠️ | Sometimes stuck on downloading resources, can be fixed by restarting the app. Overall gameplay, got stable FPS using Nvidia
 | Bad Piggies HD | 2.4.3141 | 11 | ✅
 | BanG Dream! Girls Band Party! | 4.5.0 | 11 | 🆖 | Requires GMS
 | Battle Cats Quest | 1.0.4 | 11 | ✅
 | BBC iPlayer | 4.137.0.25403 | 11 | ✅ | Sideloaded
 | Berry Browser | 3.57.8 | 11, 12 | ✅
 | Binance | 2.36.5 | 11 | ✅
-| Blue Archive (GB) | 1.36.120365 | 11 | ❌ | Black screen on app launch
-| Blue Archive (KR) | 1.39.146794 | 12, 11 | ❌ | HEVC codec support required
+| Blue Archive (GB) | 1.39.146794 | 11, 12 | ❌ | Black screen on app launch
```
Comment (pratyakshm) [inline README.md:109]:
```suggestion
| Blue Archive (GB) | 1.39.146794 | 12, 11 | ❌ | Black screen on app launch
```

File: README.md [hunk 5]
```diff
@@ -94,20 +94,20 @@ wsa://com.android.settings
...
+| Blue Archive (KR) | 1.39.146794 | 11, 12 | ❌ | HEVC codec support required
```
Comment (pratyakshm) [inline README.md:110]:
```suggestion
| Blue Archive (KR) | 1.39.146794 | 12, 11| ❌ | HEVC codec support required
```

File: README.md [hunk 6]
```diff
@@ -151,9 +151,9 @@ wsa://com.android.settings
 | Facebook Messenger | 330.0.0.12.116 (x86_64) | 11 | ⚠️ | Chat Heads don't work
 | Fancade | 1.7.6 | 11 | ❌ | App crashes
 | FAST Speed Test | 1.0.8 (88) | 11 | ✅
-| Fate/Grand Order (US) FGO | 2.22.1 (135) | 11 | ✅ || A little unstable, but playable
+| Fate/Grand Order (US) FGO | 2.34.0 | 11, 12 | 🆖 || Require Google Play Services, can be skippable if you have Google Play Service (APK) installed
 | Files by Google | Unknown | 11 | ✅ || Works fine
-| Fire Emblem Heroes | 6.0.0 | 11 | 🆖 | Requires GMS. If GMS is installed, it cannot be played due to SafetyNet error.
+| Fire Emblem Heroes | 6.7.0 | 11, 12 | 🆖 | Requires GMS. If GMS is installed, it cannot be played due to SafetyNet error.
```
Comment (pratyakshm) [inline README.md:156]:
```suggestion
| Fire Emblem Heroes | 6.7.0 | 12, 11 | 🆖 | Requires GMS. If GMS is installed, it cannot be played due to SafetyNet error.
```

File: README.md [hunk 7]
```diff
@@ -171,7 +171,7 @@ wsa://com.android.settings
 | Geekbench |5.4.1| 11 | ✅
 | GeoGebra | 5.0.674.0 | 11 | ✅
 | Geometry Dash | 2.11 | 11 | ✅ | If you use high refresh rate monitor, there is a small period where the game speeds up before the level plays for the first time and the audio will get desynced. You can simply pause and resume or die once to fix it since it won't happen on second attempt.
-| Girls' Frontline (EN) | 2.0702_362 (362) | 11 | ⚠️ || Game freezes after splash screen, works after installing WSL2, using `wsl --install`
+| Girls' Frontline (EN) | 2.0900_375 | 11, 12 | ⚠️ || Sometimes freeze while downloading resources, fixed by restarting the app
```
Comment (pratyakshm) [inline README.md:174]:
```suggestion
| Girls' Frontline (EN) | 2.0900_375 | 12, 11 | ⚠️ || Sometimes freeze while downloading resources, fixed by restarting the app
```

File: README.md [hunk 8]
```diff
@@ -365,6 +365,7 @@ wsa://com.android.settings
 | TachiyomiAZ | 8.7.0-AZ | 12, 11 | ✅
 | TachiyomiJ2K/TachiJ2K | 1.5.5 | 12, 11 | ✅ | Parsing links (from a browser) causes to open the Tachiyomi extension window or app picker dialog instead of opening TachiJ2K itself.
 | TachiyomiSY | 1.8.3 | 12, 11 | ✅
+| Tap Tap | 3.1.1 | 11, 12 | ✅ | Sometimes freeze if you brute force the app, fixed by restarting the app
```
Comment (pratyakshm) [inline README.md:368]:
```suggestion
| Tap Tap | 3.1.1 | 12, 11 | ✅ | Sometimes freeze if you brute force the app, fixed by restarting the app
```

**Review by pratyakshm — CHANGES_REQUESTED** [review]

**Comment by pratyakshm** [comment]
@ReikutsuNeko From the next time, please ensure that the Android version value is filled up in descending order (e.g.: 12, 11) and so on. Thanks! ❤️

**Comment by ReikutsuNeko** [comment]
Okay!! Thank you 😊

**Pull request merged**